src/views/landingPage/Index.jsx

```diff
@@ -81,13 +146,31 @@ const LandingPage = (props) => {
               }
             </CardColumns>
             <div>
-              <Button
-                onClick={() => getMoreTrending()} // eslint-disable-line react/jsx-no-bind
-                className="readMore"
-                variant="outline-secondary"
-                size="lg" block>
-                <h4>See more trending articles</h4>
-              </Button>
+              {
+                searchResults.searchRequest
+                && (searchResults.articlesCount > 6)
+                && ((searchResults.articles.length % 6) === 0) ? (
+                  <Button
+                    onClick={() => getMoreSearchResults()} // eslint-disable-line react/jsx-no-bind
```

Comment:
@youngestdj I made mention of avoiding these inline ESLint disabling comments. If for any reason we're gonna do this, let's do it in the ESLint config

Reply:
Fixed